src/store/actions/authentication.js

```diff
@@ -0,0 +1,54 @@
+import swal from 'sweetalert';
+
+function showAlert(text, icon) {
+  swal({
+    title: 'Authors Haven',
+    text,
+    icon,
+    button: 'Okay!',
+  });
+}
+export const requestReset = email => {
+  const mydata = {
+    email,
+  };
+  fetch('http://127.0.0.1:8000/api/password_reset/', {
+    method: 'POST',
+    headers: {
+      'content-type': 'application/json',
+    },
+    body: JSON.stringify(mydata),
+  })
+    .then(res => res.json())
+    .then(data => {
+      const text = data.message;
+      if (data.status === 1) {
+        const icon = 'success';
```

Comment:
It's better to keep the side effects of your actions very minimal, that is actions shouldn't know how to show an alert message, instead, they should dispatch other actions (Make an announcement) and the concerned parties like the `reducers` and the `Alert` component could take it from there... I suggest you use `react-toast` instead of `sweetalert`  for notifications since @nakatuddesuzan  is already it. it would be better if all our notifications were uniform.

Reply:
This makes sense! I will resort to `react-toast` and dispatch my payload to the reducer, and then pass it to the components for triggering alerts